index.html

```diff
@@ -0,0 +1,179 @@
+            </div>
+          </div>
+        </nav>
+        <div class="inner">
+          <span>"Hello! Sharing world"</span>
+          <h1>
+            creative commons <br />
+            global summit <br />
+            2022
+          </h1>
+          <div class="box-desc">
+            <p>
+              A joy celebration beliving in the value of openness and sharing,
+              creating <br />
+              a postive change with people over 80 countries is taking place in
+              October, in korea
+            </p>
+          </div>
+          <div class="info">
+            <p class="history">2022.10.15(THU) ~ 16(FRI)</p>
+            <p class="copy-right">
+              @ National Museum of korea, Art Center Nabi and more
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <!-- main programe section -->
+      <section class="main-programe">
+        <h3 class="sec-title">Main Programe</h3>
+        <article class="cards-group">
+          <div class="card">
+            <img src="./images/home/research.svg" alt="icon" />
+            <h4 class="card-title">Lecture</h4>
+            <p class="card-desc">
+              listen to the speakers from various countries about the message of
+              sharing and opening.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/home/design.svg" alt="icon" />
+            <h4 class="card-title">CC Exhibition</h4>
+            <p class="card-desc">
+              Appreciate various creations applying CC licence of
+              artists,organized from Art center Nabi.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/home/networking.svg" alt="network-icon" />
+            <h4 class="card-title">Forum</h4>
+            <p class="card-desc">
+              Have the time to share your thoughts and opinions with experts for
+              each topic.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/home/discussion.svg" alt="icon" />
+            <h4 class="card-title">Workshop</h4>
+            <p class="card-desc">
+              Try creating your own work using open source licence rather than
+              just watching it.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/home/knowledge.svg" alt="icon" />
+            <h4 class="card-title">CC Ignite</h4>
+            <p class="card-desc">
+              Get opportuntities to network with CC affiliates around the world,
+              also after the summit.
+            </p>
+          </div>
+        </article>
+        <a class="more" href="#">See the whole programe</a>
+      </section>
+      <section class="feature-section">
+        <h3 class="sec-title">Feature speakers</h3>
+        <ul class="speak-group"></ul>
+      </section>
+
+      <!-- partner section -->
+      <section class="partners">
```

Comment:
- [ ] You're missing a feature here. The `more` button. Though the interactivity is optional, but you need to implement the button design here 👍 

## DesignL
![image](https://user-images.githubusercontent.com/50990925/232543710-2b580ff3-3645-478a-8c44-2291f1e44956.png)

## Your view 👍 
![image](https://user-images.githubusercontent.com/50990925/232543795-cae1a864-bd88-4c0d-8b60-68e5d998214e.png)